bin/download-document.js

```diff
@@ -2,22 +2,30 @@
 
 /**
  * to run download-document applet from the project root folder type in your console:
- * > node bin/download-document <clienе_id> <client_secret> <username> <password> <document_id> <path_to_save> <with_history>
+ * > node bin/download-document <clienе_id> <client_secret> <username> <password> <document_id> <path_to_save>
  * <client_id>, <client_secret>, <username>, <password>, <document_id>, <path_to_save> - are required params
- * <with_history> - optional param
+ * options flags:
+ * --zip - document will be archived in zip
+ * --with-history - document will be downloaded with its history
  */
 
 'use strict';
 
+const args = process.argv.slice(2);
+const flags = args.filter(arg => /^--/.test(arg));
+const params = args.filter(arg => !/^--/.test(arg));
+
 const [
   clientId,
   clientSecret,
   username,
   password,
   documentId,
   pathToSaveFile,
-  withHistory,
-] = process.argv.slice(2);
+] = params;
+
+const zip = flags.includes('--zip');
```

Comment:
```suggestion
const withAttachments = flags.includes('--with-attachments');
```

Reply:
IMO `withAttachments` not as clear as `zip`... but if you think so, it's OK.